defaults/main.yml

```diff
@@ -12,6 +12,7 @@ solr_service_name: solr
 solr_install_dir: /opt
 solr_install_path: "/opt/{{ solr_service_name }}"
 solr_home: /var/solr
+solr_ip: "127.0.0.1"
```

Comment:
Can you change the variable name to `solr_host` instead, and default it to `localhost`? That way it maintains backwards compatibility, and can also just be a hostname like `solr.example.com`.